samples/apps/cap/README.md

```diff
@@ -0,0 +1,24 @@
+# Composable Actor Platform (CAP) for Autogen
```

Comment:
Add details explain CAP.
Autogen -> AutoGen.

Reply:
Added some details.  Let me know if you are looking for more.